faq/domains-and-dns.mdx

```diff
@@ -0,0 +1,110 @@
+- review their registrar's current registration agreement, along with any updates.
+- assume sole responsibility for the registration and use of their domain name.
+- provide accurate information for publication in directories such as WHOIS, and promptly update this to reflect any changes.
+- respond to inquiries from their Registrar within fifteen (15) days, and keep their Registrar account data and their payment information up-to-date. 
+
+## Can I modify my domain name once it has been registered?
+
+It is not possible to change domain name once the registration has been carried out. You can of course make a new registration with the correctly spelled domain name. 
+
+## How long is the registration of a domain name valid for?
+
+Your reservation is valid for a minimum of one year. You can reserve certain extensions for up to 10 years.  
+
+## What  do I do if the domain name I want to register is not available?
+
+If a domain name is unavailable, you can test a light variation on the name desired. You can also add a letter, or a number. As a last resort, contact us(lien).
+
+## Is domain renewal automatic?
+
+No. You must renew your domain name before the end of its validity period.
+If you choose the “auto-renew” feature, your domain name will be automatically renewed. 
+
+## Do you send reminders on approach of expiry?
+
+You will be notified by email 60 days before your domain name expires. Your domain name will then switch to “late renew” status. 
+
+## Why has my current registrar opposed my request for change of domain name registrar?
+
+The reasons for refusal are generally the following:
+- The domain name is locked by a status “client transfer prohibited”: the applicant forgot to remove the padlock from the registrar before requesting the transfer of domain name
+- The domain name was registered for less than 60 days
+- The domain name was the subject of a previous transfer for less than 60 days
+- The domain name has expired
+- The domain name is the subject of a dispute in progress (udrp procedure for example)
+- The owner notified his refusal to transfer the domain name to another registrar
+
+
+## How long does a domain name transfer take?
+
+A domain name transfer usually takes between 1 to 10 days. A domain name transfer can however take longer.
+
+
+## Which extensions are available during the Public beta? 
+
+There are 7 extensions available:
+- `.com` 
+- `.eu`
+- `.fr`
+- `.net` 
+- `.nl`
+- `.org`
+- `.pl` 
+
+<Message type="note">
+
+To be able to use the `.eu` extension, you need to be a resident of the European Union. 
+
+</Message>
+
+
+## Why is my domain registration refused? 
+Your domain registration may be refused if you do not meet the conditions set by the registry. 
+Ex: If your extension is “.fr” you must be a resident in France, otherwise your domain registration will be refused. 
```

Comment:
```suggestion
Ex: If your extension is “.fr” you must have an address in the European Economic Area and/or Switzerland, otherwise your domain registration will be refused. 
```